src/js/angular/controllers.js

```diff
@@ -157,10 +157,12 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
 
     $rootScope.$on('guideReset', function () {
         $scope.guidePaused = false;
+        $rootScope.guidePaused = false;
```

Comment:
Add `$rootScope.guidePaused = false;` in function resumeGuide above.

Reply:
Fixed.